src/tasks.rs

```diff
@@ -214,7 +214,7 @@ pub fn generate_spritesheet_task(
     })
 }
 
-fn config_to_camera(cam: config::PresetCamera) -> Camera {
+pub fn config_to_camera(cam: config::PresetCamera) -> Camera {
```

Comment:
Instead of making this `pub`, could you copy and paste the function into `spritec_binding`? This was kept private because it's supposed to be an internal implementation detail of `tasks`.

Also, it's not great to rely on the `config` module for any GUI stuff. We want to make it easy to remove config later as we migrate more towards the GUI. It's better to duplicate the logic in the GUI crates than to couple the GUI and the config. Copying/pasting will also allow you to customize things as you need to going forward.

Reply:
Sounds good, will be removed when the camera is implemented on the GUI side

Reply:
Thanks!